frontend/src/components/EvaluationResponses.js

```diff
@@ -72,7 +73,7 @@ const CourseModalEvaluations = ({ crn, info }) => {
 
   return (
     <div>
-      <Tabs
+      <StyledTabs
```

Comment:
These tabs dont look great in dark mode - the autogenerated option from darkreader seems nicer (screenshots below)
1. tab color currently looks kinda out of place
2. the question text shouldn't be blue
3. the question responses and separating lines should be a grey and not as bright as they currently are

![image](https://user-images.githubusercontent.com/12566801/99178289-9ab44500-26df-11eb-91b2-9800711bf945.png)

![image](https://user-images.githubusercontent.com/12566801/99178265-427d4300-26df-11eb-8e0a-fb99ec6e8973.png)